bundles/org.openhab.binding.lgthinq/src/test/java/org/openhab/binding/lgthinq/handler/LGBridgeTests.java

```diff
@@ -0,0 +1,383 @@
+    // .getConfigAs(any(Class.class));
+    // spyBridge.initialize();
+    // LGApiClientService service1 = LGApiV1ClientServiceImpl.getInstance();
+    // LGApiClientService service2 = LGApiV2ClientServiceImpl.getInstance();
+    // TokenManager tokenManager = TokenManager.getInstance();
+    // try {
+    // if (!tokenManager.isOauthTokenRegistered(fakeBridgeName)) {
+    // tokenManager.oauthFirstRegistration(fakeBridgeName, fakeLanguage,
+    // fakeCountry, fakeUserName, fakePassword);
+    // }
+    // String workId = service1.startMonitor(fakeBridgeName, "d27bdb00-7149-11d3-80b0-7440be92ac08");
+    // ACSnapShot shot = service1.getMonitorData("fakeBridgeId", "d27bdb00-7149-11d3-80b0-7440be92ac08", workId);
+    // System.out.println(shot);
+    // // service1.stopMonitor("d27cc560-7149-11d3-80b6-7440bec3653e", "n-d27cc560-7149-11d3-80b6-7440bec3653e");
+    // // String workId = service1.startMonitor("d27cc560-7149-11d3-80b6-7440bec3653e");
+    // // String workId = "n-d27cc560-7149-11d3-80b6-7440bec3653e";
+    // // service1.getMonitorData("d27cc560-7149-11d3-80b6-7440bec3653e", workId);
+    // // Thread.sleep(1000);
+    // // service1.getMonitorData("d27cc560-7149-11d3-80b6-7440bec3653e", workId);
+    // // Thread.sleep(1000);
+    // // service1.getMonitorData("d27cc560-7149-11d3-80b6-7440bec3653e", workId);
+    // // ACSnapShot ac = service1.getAcDeviceData("d7ee2251-e4bb-14a8-9d96-60ab14f3c836");
+    // // boolean ok = service1.turnDevicePower("d7ee2251-e4bb-14a8-9d96-60ab14f3c836",
+    // // DevicePowerState.DV_POWER_ON);
+    // // boolean ok = service1.changeOperationMode("d7ee2251-e4bb-14a8-9d96-60ab14f3c836", ACOpMode.COOL);
+    // // ACSnapShot ac = service1.getAcDeviceData("d7ee2251-e4bb-14a8-9d96-60ab14f3c836");
+    // // service1.changeFanSpeed("d7ee2251-e4bb-14a8-9d96-60ab14f3c836", ACFanSpeed.F3);
+    // // service1.changeTargetTemperature("d7ee2251-e4bb-14a8-9d96-60ab14f3c836", ACTargetTmp._21);
+    // List<LGDevice> devices = service1.listAccountDevices("bridgeTest");
+    // // devices.forEach((d) -> {
+    // // try {
+    // // if (d.getPlatformType().equals(PLATFORM_TYPE_V1)) {
+    // // ACCapability ac = service1.getDeviceCapability(d.getDeviceId(), d.getModelJsonUri(), true);
+    // // System.out.println(ac);
+    // // } else {
+    // // ACCapability ac = service2.getDeviceCapability(d.getDeviceId(), d.getModelJsonUri(), true);
+    // // System.out.println(ac);
+    // // }
+    // // } catch (LGApiException e) {
+    // // logger.error("Error getting capabilities", e);
+    // // }
+    // // });
+    // // service1.changeOperationMode("d27cc560-7149-11d3-80b6-7440bec3653e", ACOpMode.FAN);
+    // System.out.println("AC power ON:");
+    // } catch (LGThinqException e) {
+    // logger.error("Error testing facade", e);
+    // // } catch (InterruptedException | IOException e) {
+    // // e.printStackTrace();
+    // }
+    // }
+
```

Comment:
Please remove unused comments

Reply:
Done. I'm still changing the test to cover better the code. But I will keep it clean before update the PR's.